bundle/Resources/config/services.yml

```diff
@@ -315,6 +317,14 @@ services:
     ezrepoforms.user_register.registration_group_loader.configurable:
         class: EzSystems\RepositoryForms\UserRegister\ConfigurableRegistrationGroupLoader
         arguments:
-            - @ezpublish.api.repository
+            - "@ezpublish.api.repository"
         calls:
             - [setGroupId, ["$users.registration_group$"]]
+
+    ezrepoforms.form_data_mapper.user_register:
+        class: EzSystems\RepositoryForms\Data\Mapper\UserRegisterMapper
+        arguments:
+            - "@ezpublish.api.repository"
```

Comment:
Reason is Sf 3.0 compatibility. We will start getting deprecation notices unless we add those quotes. You should actually do the same from now on :-)